src/components/dialogs/EditPerson.tsx

```diff
@@ -97,6 +97,7 @@ const EditPersonDialog = ({
         subjectGroup: person.role == "teacher" ? person.subjectGroup.id : 0,
         email: person.contacts.filter((contact) => contact.type == "Email")[0]
           ?.value,
+        isAdmin: person.isAdmin || false,
```

Comment:
using `??` here would make more sense since `person.isAdmin` is already a boolean variable and `||` will check for anything falsy